packages/mobile/locales/en-US/onboarding.json

```diff
@@ -4,12 +4,9 @@
     "body": "Adding your contacts makes it easy to send and request payments with your friends. Connecting does not send invites to your contacts.",
     "findSwitch": "Allow contacts to find you",
     "syncing": {
-      "loadingHeader": "Syncing contacts...",
-      "loadingStatus0": "Importing",
-      "loadingStatus1": "Processing {{total}} contacts",
-      "loadingStatus2": "Finding matches",
```

Comment:
Why have we removed these? For slow phones, having more steps would be nice to show progress?

Reply:
Discussed with @coreycelo. I do not feel strongly about this.

Reply:
My understanding is we can no longer show progress (ie. 30 of 134 contacts) and only total (134 contacts) so we simplified it since there was less value add.

Reply:
The importing step is while we literally import and the finding step is while we wait for matchmaking results from PGPNP. Given that they're distinct tasks and I already did the work to dispatch fine-grained status updates, I don't know why we would cut this.

Reply:
Designs have been updated and I've added it back in.

Reply:
Yeah on most phones it'll be too fast. On my old Nexus 7 with 2000 contacts it shows for a while.